includes/reader-revenue/woocommerce/class-woocommerce-connection.php

```diff
@@ -1182,32 +1182,57 @@ public static function send_customizable_receipt_email( $enable, $order, $class
 			return $enable;
 		}
 
-		$currency = $order->get_currency();
-		$symbol   = newspack_get_currency_symbol( $currency );
-		$total    = $order->get_total();
-		$created  = $order->get_date_created();
+		$frequencies = [
+			'month' => __( 'Monthly', 'newspack-plugin' ),
+			'year'  => __( 'Yearly', 'newspack-plugin' ),
+		];
+		$product_map = [];
+		foreach ( $frequencies as $frequency => $label ) {
+			$product_id = Donations::get_donation_product( $frequency );
+			if ( $product_id ) {
+				$product_map[ $product_id ] = $label;
+			}
+		}
+
+		$item = array_values( $order->get_items() )[0];
```

Comment:
tip: `array_shift` is also an alternative here

Reply:
Updated in e2cb1073e3886930806c53d18e53c7817a7d8d6a